charts/apisix-ingress-controller/templates/configmap.yaml

```diff
@@ -31,12 +41,23 @@ data:
       - {{ . | quote }}
       {{- end }}
       {{- end }}
+
+      {{- if gt (len .Values.config.kubernetes.appNamespaces) 0 }}
```

Comment:
Sorry, it looks that here should be ".Values.config.kubernetes.namespaceSelector". 😂